contracts/lib/local/LocalCallDataParserLib.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.26;
+
+library LocalCallDataParserLib {
```

Comment:
why 'Local' ?

Reply:
Because it is not universal. And most probably not reusable. It is just for the sake of code readability.